cas-server-core-tickets/src/main/java/org/jasig/cas/ticket/ProxyGrantingTicketImpl.java

```diff
@@ -19,11 +22,20 @@
  * @author Misagh Moayyed
  * @since 4.1
  */
+@Entity
+@DiscriminatorValue("PGT")
 public final class ProxyGrantingTicketImpl extends TicketGrantingTicketImpl implements ProxyGrantingTicket {
     private static final long serialVersionUID = -8126909926138945649L;
 
     /**
      * Instantiates a new proxy granting ticket impl.
+     */
+    public ProxyGrantingTicketImpl() {
+        // nothing to do
```

Comment:
it's already being called by default because no other `super()` or `this()` statement is present, so it's just a question of style.

Let me know if you want it in there stylistically.